ui/app/templates/components/shamir-modal-flow.hbs

```diff
@@ -131,24 +146,24 @@
           {{/if}}
           <div class="box is-shadowless is-marginless no-padding-top is-fullwidth no-padding-sides" data-test-form-text>
             {{#if otp}}
-              <div>
-                <h4 class="title is-7 has-bottom-padding-m is-fullwidth">
+              <div class="has-bottom-margin-xxl">
+                <h4 class="field-title">
                   One-time password (OTP)
                 </h4>
+                <p class="help has-text-grey has-bottom-margin-xs">
+                  This OTP will be used to encode the generated operation token. <span class="has-text-semibold">Save this</span>, as you will need it later to decode the operation token.
```

Comment:
nit: technically, this should be "decode the generated operation token", just like the instructions here: https://github.com/hashicorp/vault/pull/9355/files#diff-0736a8f910677faecdbf8bfb0178ed3eR32

Reply:
Whoops, that was copypasta from the mockups. Good catch!